src/BlockTypes/ProductBestSellers.php

```diff
@@ -19,6 +19,14 @@ class ProductBestSellers extends AbstractProductGrid {
 	 * @param array $query_args Query args.
 	 */
 	protected function set_block_query_args( &$query_args ) {
-		$query_args['orderby'] = 'popularity';
```

Comment:
This is the [correct query](https://github.com/woocommerce/woocommerce/blob/c4f00719de39b38257d66cc2e171465ea7c4e9db/plugins/woocommerce/includes/class-wc-query.php#L730-L734) for best selling product. I wonder why do you need to change it 🤔